nicegui/ui_run.py

```diff
@@ -121,10 +121,11 @@ def run(*,
     if native:
         show = False
         host = host or '127.0.0.1'
-        port = native_module.find_open_port()
+        port = port or native_module.find_open_port()
         width, height = window_size or (800, 600)
         native_module.activate(host, port, title, width, height, fullscreen, frameless)
     else:
+        port = 8080     
```

Comment:
How can a user define the port when not in native mode? It seems to always be set to 8080.

Reply:
I changed it to
```python
port = port or 8080
```